src/edit/ConfirmationDialog.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <v-dialog v-model="dialog" width="500">
```

Comment:
This is replaced with TailorDialog inside: https://github.com/ExtensionEngine/tce-sprout-video/pull/3/files#diff-fd06d3d1b05de79c059e83f698f712fe09fd734f38775a9de1d261ea1a8766f0R1